Remove yay from readme, remove outdated warning (#407)
rocm-arch/rocm-arch: README.md

# ROCm for Arch Linux
This repository hosts a collection of [Arch Linux](https://www.archlinux.org/) [PKGBUILDs](https://wiki.archlinux.org/index.php/PKGBUILD) for the [AMD ROCm Platform](https://rocmdocs.amd.com/en/latest/).
These scripts implement a great portion of the stack, ranging from low level interfaces, over compilers to high-level application librariers.

## Installation
The Arch Linux packages for ROCm are available on the [AUR](https://wiki.archlinux.org/index.php/Arch_User_Repository).

> Since many packages will be installed, we recommend to use the [`arch4edu`](https://wiki.archlinux.org/index.php/Unofficial_user_repositories#arch4edu) binary repository as it will

Reviewer: I think we should keep the `yay` instructions. We are recommending to use arch4edu repo, already and I think keeping yay covers packages that are not in arch4edu. Until all packages in this repo are in arch4edu I do not feel comfortable changing this.

Reviewer: From a developer's standpoint, I like people that use yay or makepkg since that means more bugs get reported and issues found.

Author: You're right. The last point is very important as we've seen in #406

greatly speed up your installation time.
For directions see [Add arch4edu to your Archlinux](https://github.com/arch4edu/arch4edu/wiki/Add-arch4edu-to-your-Archlinux).

To install ROCm, use
```bash

> # pacman -Syu rocm-dev rocm-utils rocm-libs

Reviewer: While people do `-Syu` all the time, `-S` is the only flag need to install.
I like the change to `rocm-dev rocm-utils rocm-libs` tho!

Author: Sorry for being sloppy here.

```

You can also install specific ROCm packages like so:
```bash

> # pacman -S rocminfo

Reviewer: Ah one more thing I don't like using shell prefixes like `#` and `$` because then you can't just copy paste the line.

Author: Yeah, people should know that you install packages with root privileges.

```

For additional installation configuration, such as adding a user to the `video`
group, we refer to AMD's [installation guide](https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html).

To uninstall, use the following command:
```bash
# pacman -R rocm-dev rocm-utils rocm-libs
```

For more helpful tips see the ArchWiki entry on ROCm, [here](https://wiki.archlinux.org/index.php/GPGPU#ROCm).

## Contributing
Your contribution is always welcome. You can use the [issue tracker](https://github.com/rocm-arch/rocm-arch/issues) to report problems with the AUR packages.
Optimally, you open a pull request that solves your problem.

> For most packages, you have to update `pkgver` and `sha256sums`. Before you commit your changes please generate `.SRCINFO` from the updated `PKGBUILD`:

Reviewer: maybe instead of `changes please generate` let's do `changes you will need to generate` to make it seem that is is required (which it is)

# ROCm for Arch Linux
This repository hosts a collection of [Arch Linux](https://www.archlinux.org/) [PKGBUILDs](https://wiki.archlinux.org/index.php/PKGBUILD) for the [AMD ROCm Platform](https://rocmdocs.amd.com/en/latest/).
These scripts implement a great portion of the stack, ranging from low-level interfaces, over compilers to high-level application librariers.

## Installation
The Arch Linux packages for ROCm are available on the [AUR](https://wiki.archlinux.org/index.php/Arch_User_Repository).
Since many packages will be installed, we recommend to use an [AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers) like [`yay`](https://aur.archlinux.org/packages/yay/).

> Binary versions are hosted by [`arch4edu`](https://wiki.archlinux.org/index.php/Unofficial_user_repositories#arch4edu). Using them will greatly speed up your installation time.

Reviewer: Let's keep the same wording as before since in this version we are not recommending arch4edu.

Reviewer: ```suggestion
Since many packages will be installed, it is recommended to use an [AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers)
like [`yay`](https://aur.archlinux.org/packages/yay/).

It is also recommended to use the [`arch4edu`](https://wiki.archlinux.org/index.php/Unofficial_user_repositories#arch4edu) binary repository as it will
greatly speed up your installation time.
```

For directions see [Add arch4edu to your Archlinux](https://github.com/arch4edu/arch4edu/wiki/Add-arch4edu-to-your-Archlinux).

To install ROCm, use
```bash
yay -S rocm-dev rocm-utils rocm-libs
```
which includes the low-level components and compilers (`rocm-dev`), utilities like `rocminfo` (`rocm-utils`) and GPU-accelerated math libraries (`rocm-libs').

You can also install specific ROCm packages like so:
```bash
yay -S rocminfo
```

For additional installation configuration, such as adding a user to the `video`
group, we refer to AMD's [installation guide](https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html).

To uninstall, use the following command:
```bash
yay -R rocm-dev rocm-utils rocm-libs
```

For more helpful tips see the ArchWiki entry on ROCm, [here](https://wiki.archlinux.org/index.php/GPGPU#ROCm).

## Contributing
Your contribution is always welcome. Use the [issue tracker](https://github.com/rocm-arch/rocm-arch/issues) to report problems with the AUR packages.
Optimally, you open a pull request that solves your problem.

> For most packages, you have to update `pkgver` and `sha256sums`. Before you commit your changes you will need generate `.SRCINFO` from the updated `PKGBUILD`:

Reviewer: ```suggestion
For most packages, you have to update `pkgver` and `sha256sums`. Before you commit your changes you will need to generate `.SRCINFO` from the updated `PKGBUILD`:
```

Author: Thanks for spotting this.
